libretro/libretro.c

```diff
@@ -3918,18 +3965,24 @@ void retro_run(void)
       {
          thisframe_y_adjust_update_frame_timer--;
          if (thisframe_y_adjust_update_frame_timer == 0)
+         {
             if (opt_vertical_offset != 0)
+            {
                thisframe_y_adjust = minfirstline + opt_vertical_offset;
+               request_reset_drawing = true;
+            }
+         }
```

Comment:
This can just be:

```c
if (thisframe_y_adjust_update_frame_timer == 0 && opt_vertical_offset != 0)
{
   thisframe_y_adjust = minfirstline + opt_vertical_offset;
   request_reset_drawing = true;
}
```

Reply:
Another good one! I'll fix that :)

Reply:
Done!